NOTICE

```diff
@@ -1,4 +1,15 @@
 CosmWasm-Plus: A collection of production-quality CosmWasm contracts
 
-Different crates in this repo may be either Apache-2.0 or AGPL-3.0 licensed.
-Please refer to the NOTICE file in each crate (and the Cargo.toml metadata)
\ No newline at end of file
+Copyright 2021 Confio GmbH
```

Comment:
Is it?

Reply:
What do you mean? I think the idea with these attributions is to let people know whose intellectual property this is, but I never dived deep into that.

Reply:
I was wondering if project being under CosmWasm org isn't somehow under it as well from copyright standpoint - in previous license there copyright wasn't mentioned at all..
But as an intellectual property attribute, sure - I see no issues.

Reply:
Ah, I see! I figured Confio is the only legal entity we have, and in the [`cosmwasm` repo we state the company as the copyright holder](https://github.com/CosmWasm/cosmwasm/blob/main/NOTICE#L4).

@webmaster128 what do you think?

Reply:
The whole point here is to list the copyright holders, which then at the same time grant a license to use the software. Strictly speaking every freelancer who's not employed could be listed individually. The list might not be exhaustive, but its a good start. More elements might be added as in the cosmwasm example.

The only change I'd do for now is change the year to a list ("2021,2022 Confio GmbH").